### ISSUE TITLE
Animations are sluggish

### ISSUE THREAD

**Issue by ltGuillaume** [msoultanidis/quillnote#51]
I see that the animations used in the app are very costly and seem to be sluggisch quite often. Perhaps this could be optimized or optionally disabled for older devices?

**Comment by msoultanidis** [comment]
Are all animations sluggish or only the animation that plays when you click on a note?

**Comment by ltGuillaume** [comment]
Yes, the one when opening a note. Sorry, should have more specific.

**Comment by matm** [comment]
Sluggish even on fast devices like Redmi Note 7. Opening / exiting a note is a slow animation (low frame rate).

**Comment by yant0** [comment]
2nd this, i use galaxy core 2. its crippling
